packages/ramda-extension/src/__tests__/isBoolean-test.js

```diff
@@ -0,0 +1,33 @@
+import { any, equals } from 'ramda';
+import { isBoolean } from '../';
+
+describe('isBoolean', () => {
+	it('should exist', () => {
+		expect(isBoolean).toBeTruthy();
+	});
+
+	it('returns true for Boolean type', () => {
+		expect(any(equals(true), [
```

Comment:
Shouldn't all equals true?

Reply:
My mistake was in hurry, will fix. I inspired with isNumber-test.js, that will need fix as well.

Reply:
@jbradle updated